### PR TITLE
Add Popup Window

### PR DESCRIPTION
**Pull request by David-Lanzz**
- We have added a new popup window for the portfolio cards
- We made the popup window responsive to screen size using CSS and javascript
- We have solved all linter errors

### CODE REVIEW

**Review by TadesseDev — COMMENTED** [review]
### Hi @David-Lanzz 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

You have submitted a project that has linter tests falling. Please make sure all linters are set up and passing before your next review request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -70,14 +79,66 @@ <h3>Tonic</h3>
...
+                        <li class="js">javaScript</li>
+                    </ul>
+                    <ul>
+                        <li class="popupBtn">
+                            <div>
+                                <a href="#" class="buttons">See live<img src="./images/Icon.png" alt="github icon"></a>
+                            </div>
+                            <div> <a href="#" class="buttons">See Source<img src="./images/Vector22.png"
+                                        alt="github icon"></a>
+                            </div>
```
Comment (brainconnect93) [inline index.html:130]: - [ ] Hi @David-Lanzz, well done working on the pop-up section of your project, 👍 👍 but according to the project requirement, it is paramount that the pop-up is created dynamically and the hardcoded should be removed from the HTML file. I would kindly suggest you create the pop-up dynamically and remove the code you have in the HTML file to see if the dynamic function works fine in your JavaScript file 👍 

  - N.B: The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.** Check an example of a student's projects: [Link 1](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and [Link 2](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96)

File: index.html [hunk 2]
```diff
@@ -107,14 +168,14 @@ <h3>Uber Navigation</h3>
...
+                    <a href="#second-button" class="buttons">
+                        <h3 id="second-button">See Project</h3>
                     </a>
                 </li>
             </ul>
         </div>
-        <div class="projects">
-            <img class="img" src="./images/Snapshoot Portfolio 3.png" alt="Project Image">
+        <div class="projects" id="thirdCard">
+            <img class="img" src="./images/third.png" alt="Project Image">
```
Comment (brainconnect93) [inline index.html:178]: - [ ] Hi @David-Lanzz, I would kindly suggest that all the project cards should be created dynamically and all the hardcoded written in the HTML file should be removed, With this, you'll be sure you followed the project requirement as much as possible. Kindly Fix it.

  - N.B: The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.** Check an example of a student's projects: [Link 1](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and [Link 2](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96).

File: index.html [hunk 3]
```diff
@@ -144,19 +205,19 @@ <h3>Facebook 360</h3>
 
                 </li>
                 <li class="view">
-                    <a href="" class="buttons">
-                        <h3>See Project</h3>
+                    <a href="#third-button" class="buttons">
+                        <h3 id="third-button">See Project</h3>
```
Comment (brainconnect93) [inline index.html:209]: - [ ] I would kindly advise you don't use a `heading tag` as a child element to an `anchor tag` this is for semantic and conventional reasons. kindly leave it as an `anchor tag` or a `button tag` for all your `see project` button in your project cards.

- E.G:
```HTML
  <a href="#third-button" class="buttons" id="third-button">See Project</a>
```

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

Hi @David-Lanzz, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by David-Lanzz** [comment]
Hi,please your review wasn't fair, I worked with about 8 people on this
project and every single one of us created a card for the popup and changed
the content dynamically using JavaScript, asking me to delete all html
codes I worked on for a whole two days is like you intentionally want me to
repeat the week,please kindly review again and with mercy this time,
thankyou... because I do not know how I'm supposed to add all these things
using JavaScript alone

**Comment by brainconnect93** [comment]
hello @David-Lanzz, It's part of the project requirement that the project card should be created dynamically and all the hardcoded lines of codes should be removed from the HTML file. Please am sorry if I am unfair to have requested these changes but they are things I should ask you to work on Because they will always help in the long run as a web developer. Kindly let me know if there is anything I can do to help.
Thank You.

**Comment by David-Lanzz** [comment]
Well I'm sorry about my outburst,I was only frustrated at the moment,I went
through the project requirements now and I just missed that part... my
question now is,if I use the document.createElement() to create all those
html elements for the popup and use element.setAttribute() to set their
classes to the same ones I used in the hard coded popup codes, I do not
need to rewrite the js and css codes all over again right?

**Pull request merged**